coldfront/core/project/views.py

```diff
@@ -1823,13 +1831,16 @@ def get_users_to_review(project_obj):
                      delay)
             except ProjectUserJoinRequest.DoesNotExist:
                 auto_approval_time = 'Unknown'
+
+            reason = ele.projectuserjoinrequest_set.latest('created').reason
```

Comment:
This should be moved within the `try/except` block above in case there is no latest.